hack/bump-common-instancetypes.sh

```diff
@@ -2,15 +2,19 @@
 
 set -ex
 
-source $(dirname "$0")/common.sh
-source $(dirname "$0")/config.sh
+source "$(dirname "$0")/common.sh"
+source "$(dirname "$0")/config.sh"
 
-function latest_version() {
-    local org="$1"
-    local repo="$2"
+target_branch=${1:-"main"}
 
-    curl --fail -s "https://api.github.com/repos/${org}/${repo}/releases/latest" |
-        jq -r '.tag_name'
+function latest_version() {
+    if [[ $target_branch == "main" ]]; then
+        curl --fail -s "https://api.github.com/repos/kubevirt/common-instancetypes/releases/latest" |
+            jq -r '.tag_name'
+    else
+        curl --fail -s https://api.github.com/repos/kubevirt/common-instancetypes/releases |
```

Comment:
This will return only the 30 latest releases, which most likely won't be an issue, but in theory it could happen that this does return nothing that matches `.target_commitish` leading to the return of an empty string. There's no easy way to avoid this but you could add the query parameter `per_page=100` to the API call to make it a bit less likely to happen.